client/src/components/UserShowPage.js

```diff
@@ -2,110 +2,119 @@ import React, { useState } from 'react';
+    }
+  }
 
-    const getUser = async () => {
-      try {
-            const response = await fetch(`/api/v1/users/${id}`)
-            if(!response.ok) {
-              const errorMessage = `${response.status}: (${response.statusText})`
-              throw new Error(errorMessage)
-            }
-            const body = await response.json()
-            setUser(body.user)
-          } catch (error) {
-            console.error(`error in fetch: ${error}`)
-          }
-        }
+  useState(() => {
+    getUser()
+  }, [])
 
-        useState(() => {
-          getUser()
-        }, [])
+  let adminText = ""
+  if (currentUser?.is_admin) {
+    adminText = <h6>Signed in as administrator</h6>
+  }
+    
+  let dropzoneComponent = ""
+  if(currentUser?.id === user.id){
+    dropzoneComponent = (
+      <div>
+        <h3>Click below to upload image</h3>
 
-    let dropzoneComponent = ""
-    if(currentUser?.id === user.id){
-      dropzoneComponent = (
-        <div className="dropzone">
-          <h3>Click below to upload image</h3>
-          <form onSubmit={addProfileImage}>
-            <Dropzone onDrop={handleImageUpload}>
-              {({getRootProps, getInputProps}) => (
-                <section>
-                  <div {...getRootProps()}>
-                    <input type="text" {...getInputProps()} />
-                    <p>input your profile image here</p>
-                  </div>
-                </section>
-              )}
-            </Dropzone>
-            <input className='button' type='submit' value='save profile' />
-          </form>
-          <img src={uploadedImage.preview} className="profile-image-preview" />
-        </div>
-      )
-    }
+        <form onSubmit={addProfileImage}>
+          <Dropzone onDrop={handleImageUpload}>
+            {({getRootProps, getInputProps}) => (
+              <section>
+                <div {...getRootProps()}>
+                  <input type="text" {...getInputProps()} />
+                  <p>input your profile image here</p>
+                </div>
+              </section>
+            )}
+          </Dropzone>
+          <input className='button' type='submit' value='save profile' />
+        </form>
+        <img src={uploadedImage.preview} className="profile-image-preview" />
+      </div>
+    )
+  }
     
-    const DateObject = new Date(user.createdAt)
-    const createdDateString= DateObject.toUTCString()
+  const DateObject = new Date(user.createdAt)
+  const createdDateString= DateObject.toUTCString()
     
-    return ( 
-        <div className='callout'>
-          <div>
-              <h1>{user.username}'s Profile</h1>
-              <h4>{user.email}</h4>
-              <p>This user has been user since {createdDateString}</p>
-              <img src={user.profileImage} className='profile-image' alt='profile-image' />
-          </div>
-          {dropzoneComponent}
-        </div>
-    );
+  return ( 
+    <div>
+      <div>
```

Comment:
both of these divs arent doing anything. replace the outer one with a react fragment, and nuke the inner one